Parsers/ASimUserManagement/Parsers/ASimUserManagementLinuxAuthpriv.yaml

```diff
@@ -0,0 +1,340 @@
+Parser:
+  Title: User Management ASIM parser for Linux Authpriv logs
+  Version: '0.1.0'
+  LastUpdated: 4 Oct, 2023
+Product:
+  Name: Microsoft
+Normalization:
+  Schema: UserManagement
+  Version: '0.1.1'
+References:
+- Title: ASIM User Management Schema
+  Link: https://aka.ms/ASimUserManagementDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Ubuntu remote logging
+  Link: https://manpages.ubuntu.com/manpages/lunar/en/man5/rsyslog.conf.5.html 
+- Title: gpasswd
+  Link: https://manpages.ubuntu.com/manpages/lunar/en/man1/gpasswd.1.html
+- Title: groupadd
+  Link: https://manpages.ubuntu.com/manpages/lunar/en/man8/groupadd.8.html
+- Title: groupdel
+  Link: https://manpages.ubuntu.com/manpages/lunar/en/man8/groupdel.8.html
+- Title: groupmod
+  Link: https://manpages.ubuntu.com/manpages/lunar/en/man8/groupmod.8.html
+- Title: useradd
+  Link: https://manpages.ubuntu.com/manpages/lunar/en/man8/useradd.8.html
+- Title: userdel
+  Link: https://manpages.ubuntu.com/manpages/lunar/en/man8/userdel.8.html
+- Title: usermod
+  Link: https://manpages.ubuntu.com/manpages/lunar/en/man8/usermod.8.html
+Description: |
+  This ASIM parser supports normalizing Linux authpriv logs delivered using Syslog to the ASIM UserManagement normalized schema.
+ParserName: ASimUserManagementLinuxAuthpriv
+EquivalentBuiltInParser: _ASim_UserManagement_LinuxAuthpriv
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser = (
+      disabled:bool = false
+  ) {
+  let ActionLookup = datatable (Action:string, EventType:string)
+  [
+      "added",  "UserAddedToGroup",
+      "removed","UserRemovedFromGroup"
+  ];
+  let SeverityLookup = datatable (SeverityLevel:string, EventSeverity:string)
+  [
+      "info", "Informational",
+      "warn", "Low",
+      "err",  "Medium",
+      "crit", "High"
+  ];    
+  let ItemParser = (T:(SyslogMessage:string,SeverityLevel:string,ProcessID:int)) {
+      T
+      | lookup SeverityLookup on SeverityLevel
+      | extend ActingAppId = tostring(ProcessID)
+      | project-away SyslogMessage,SeverityLevel, ProcessID
+  };
+  let SyslogParsed = (
+      Syslog
+      | where not(disabled)
+      | where Computer in (_ASIM_GetSourceBySourceType('LinuxAuthpriv'))
+      | where Facility == "authpriv"
+          and ProcessName in ("useradd","usermod","userdel","groupadd","groupmod","groupdel","gpasswd")
+      | project-away EventTime,Facility,MG,CollectorHostName,SourceSystem,TenantId
+  );
+  union (
+      SyslogParsed
+      | where not(disabled)
+      | where ProcessName == "useradd"
+          and SyslogMessage startswith "new user: name="
+      | parse SyslogMessage with "new user: name=" TargetUsername ", UID=" TargetUserId ", GID=" GroupId ", " *
+      | extend 
+          EventType   = "UserCreated", 
+          EventResult = "Success"
+      | invoke ItemParser()
```

Comment:
@KustoKing 
do we need to call this every time?  can't we use it once at the end?